internal/util/paramtable/base_table.go

```diff
@@ -334,6 +336,30 @@ func (gp *BaseTable) ParseIntWithDefault(key string, defaultValue int) int {
 }
 
 // package methods
+// GetConfigurationByName get member variable from config struct by name
+func GetConfigurationByName(cfg interface{}, name string) (string, error) {
+	reflectCfg := reflect.ValueOf(cfg)
```

Comment:
Why not read the value from BaseTable directly

Reply:
Because we use the pattern name directly, so this depending on what format of name we want to use, the name in Basetable was like "queryNode.cache.enabled", and in Config struct was “CacheEnable”.